screens/main.py

```diff
@@ -60,20 +59,20 @@ def __init__(self, screen, **kwargs):
         self.objects.append(self.timeout)
         self.timeout.start()
 
-    def switch_to_screen(self, param, pos):
+    def switch_to_screen(self, param):
         from .screen_manager import ScreenManager
         screen_manager = ScreenManager.get_instance()
         screen_manager.set_active(
             NamesScreen(self.screen, param)
         )
 
     @staticmethod
-    def home(param, pos):
+    def home():
```

Comment:
this will probably blow up because the method is called with (param)

Reply:
When I was testing there were no problems, but I'll habe a look into it.